src/c_client.cpp

```diff
@@ -3,6 +3,7 @@
 
 
 using douban::mc::Client;
+using douban::mc::errCodeToString;
```

Comment:
are we need this?

Reply:
@tclh123 rm this line seems also works。but do you know why?

Reply:
okay there is some "Client" usage in c_client.cpp。